src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -846,7 +846,10 @@ typedef enum {
     RTC_CODEC_VP8 = 3,                                                            //!< VP8 video codec.
     RTC_CODEC_MULAW = 4,                                                          //!< MULAW audio codec
     RTC_CODEC_ALAW = 5,                                                           //!< ALAW audio codec
-    RTC_CODEC_UNKNOWN = 6,
+    RTC_CODEC_UNKNOWN = 8,
```

Comment:
The value is still modified for `UNKNOWN`. `UNKNOWN` needs to be 6 and the rest can follow.

Reply:
Oops, messed up while copying it down. Fixed it now. Nice catch!